generator.rb

```diff
@@ -0,0 +1,73 @@
+def is_vowel?(letter)
+  vowels = ["a", "e", "i", "o", "u"]
+  if vowels.include?(letter.downcase)
+    return true
+  end
+  return false
+end
+
+
+def get_word_from_user(word)
+  puts "What is the #{word} to combine?"
```

Comment:
`word` as a variable on its own is weird here. My own (admittedly fiddly) opinion is that it's not descriptive enough. Maybe `word_num`, or `cardinal`?